kore/src/Kore/Repl/Data.hs

```diff
@@ -446,94 +439,78 @@ helpText =
+    \       - need to be prefixed with the module name, e.g. IMP.myName\n\
+    \\nAvailable entry types:\n"
+    <> entriesForHelp
+    <>
+    "\n\
+    \(*) If an edge is labeled as Simpl/RD it means that the target node\n\
+    \ was reached using either the SMT solver or the Remove Destination step.\n\
+    \    A green node represents the proof has completed on\
     \ that respective branch. \n\
-    \ A red node represents a stuck configuration.\n\
-    \(***) An interesting branching node has at least two children which\n\
+    \    A red node represents a stuck configuration.\n\
+    \(**) An interesting branching node has at least two children which\n\
     \ contain non-bottom leaves in their subtrees. If no such node exists,\n\
     \ the current node is advanced to the (only) non-bottom leaf. If no such\n\
     \ leaf exists (i.e the proof is complete), the current node remains the same\n\
     \ and a message is emitted.\n\
-    \ (****) The clear command doesn't allow the removal of nodes which are direct\n\
+    \(***) The clear command doesn't allow the removal of nodes which are direct\n\
     \ descendants of branchings. The steps which create branchings cannot be\n\
-    \ partially redone. Therefore, if this were allowed it would result in invalid proofs.\n\
-    \\n\n\
-    \Rule names can be added in two ways:\n\
-    \    a) rule <k> ... </k> [label(myName)]\n\
-    \    b) rule [myName] : <k> ... </k>\n\
-    \Names added via a) can be used as-is. Note that names which match the\n\
-    \ indexing syntax for the try and tryf commands shouldn't be added\n\
-    \ (e.g. a5 as a rule name).\n\
-    \Names added via b) need to be prefixed with the module name followed by\n\
-    \ dot, e.g. IMP.myName\n\
-    \Available entry types:\n    "
-    <> intercalate "\n    " Log.getEntryTypesAsText
-    <> "\n\n\
-    \For logging the succesfully applied equations, attempted equations, or both,\n\
-    \launch kore-repl with the appropriate flags:\n\
-    \--debug-apply-equation EQUATION_IDENTIFIER\n\
-    \--debug-attempt-equation EQUATION_IDENTIFIER\n\
-    \--debug-equation EQUATION_IDENTIFIER;\n\
-    \For more details run: kore-repl --help\n"
+    \ partially redone. Therefore, if this were allowed it would result in invalid proofs.\n"
+    <> "\nFor more details see https://github.com/kframework/kore/wiki/Kore-REPL#available-commands-in-the-kore-repl\n"
+
+
+entriesForHelp :: String
+entriesForHelp =
+    zipWith3
+        (\e1 e2 e3 -> align e1 <> align e2 <> align e3)
+        column1
+        column2
+        column3
+    & unlines
+  where
+    entries :: [String]
+    entries =
+        Log.entryTypeReps
+        & fmap show
+        & sort
+        & (<> ["", ""])
+    align entry = entry <> replicate (40 - length entry) ' '
+    columnLength = length entries `div` 3
+    (column1And2, column3) = splitAt (2 * columnLength) entries
+    (column1, column2) = splitAt columnLength column1And2
```

Comment:
I just realized it might be more eficient to to have (column1, column2And3) first, and then split the last two